crates/rpc/src/server.rs

```diff
@@ -23,6 +33,10 @@ impl ScrollSgxServer for ScrollSgxServerImpl {
         &self,
         _req: ProveBatchRequest,
     ) -> Result<ProveBatchResponse, ErrorObjectOwned> {
+        // simple signing example
```

Comment:
I think attestation report is about to generate attestation for SGX prover, which used to register the prover in on-chain verifier.